lib/NarrativeService/apps/appinfo.py

```
@@ -1,32 +1,31 @@
+import contextlib
+
 from installed_clients.CatalogClient import Catalog
 from installed_clients.NarrativeMethodStoreClient import NarrativeMethodStore
 
 IGNORE_CATEGORIES = {"inactive", "importers", "viewers"}
+APP_TAGS = {"release", "beta", "dev"}
 
 
-def get_ignore_categories():
-    return {x: 1 for x in IGNORE_CATEGORIES}
+def get_ignore_categories() -> dict[str, int]:
+    return dict.fromkeys(IGNORE_CATEGORIES, 1)
 
 
-def _shorten_types(type_list):
+def _shorten_types(type_list: list[str]) -> list[str]:
     """
     convert ['KBaseMatrices.AmpliconMatrix'] to ['AmpliconMatrix']
     """
-    shorten_types = list()
+    shorten_types = []
     for t in type_list:
-        try:
+        with contextlib.suppress(IndexError):
```

Another piece of ruffage since we're ignoring errors anyway.